addons/im_livechat/static/src/core/common/thread_model_patch.js

```diff
@@ -69,6 +69,14 @@ patch(Thread.prototype, {
             ? _t("This livechat conversation has ended")
             : "";
     },
+
+    get canUnpin() {
+        if (this.channel_type === "livechat") {
+            return this.importantCounter === 0;
+        }
+        return super.canUnpin;
+    },
+
```

Comment:
Why this change?

Sure it wasn't consistent with "chat" behaviour to require reading the conversation before being able to unpin, but now that this uses "..." button I'd argue the livechat way to allow unpin even with unread messages is better.

So I would remove these LOCs, and put similar behaviour to chats.